nixos/modules/services/misc/portunus.nix

```diff
@@ -107,8 +107,8 @@ in
     ldap = {
       package = mkOption {
         type = types.package;
-        default = pkgs.openldap;
-        defaultText = lib.literalExpression "pkgs.openldap";
+        default = pkgs.openldap-libxcrypt-legacy;
```

Comment:
Default to the local override here rather than any top-level package.

Reply:
Also, please add a comment on the override mentioning it has to be migrated someday, please open an issue upstream if it's not planned now, this has to be done at some point. This is a upstream problem.

Reply:
> Default to the local override here rather than any top-level package.

Then the package won't be in the cache. I had that discussion a while ago with someone about another package which I forgot in the meanwhile.

Building openldap locally is not that fun because the tests run shell scripts which take a good amount of time on a normal computer.

Reply:
The software requires setup via the web ui and the API is not documented or with other words unstable. I am using the software on a regular basis which needs to be enough for now.

Reply:
That's irrelevant to adding a NixOS test that just do a curl check and will force cache all you need in Hydra :).

Reply:
I added a dead basic nixos test, I don't have time for more right now